Assignment 1/test/SimpleRegistryTest.t.sol

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "forge-std/Test.sol";
+import "src/SimpleRegistry.sol";
+
+/// @title Tests the simple name registry.
+/// @author Thomas den Hollander
+/// @notice Used to test the required functionality for the `SimpleRegistry` contract.
+contract SimpleRegistryTest is Test {
+    SimpleRegistry simpleRegistry;
+
+    constructor() {
+        simpleRegistry = new SimpleRegistry();
+    }
+
+    /// @notice Test claiming some names for this contract.
+    function testClaimNames() public {
+        assertEq(simpleRegistry.owners("TestName"), address(0));
+        simpleRegistry.claimName("TestName");
```

Comment:
Please test the event being emitted as well. We test:
- Storage changes
- Errors
- Events
- Return values
- That we call external contracts with the right parameters.

Not all of them apply here, of course.